content/blog/1-from-gantt-charts-to-generative-ai-how-agentic-ai-is-revolutionizing-project-management.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Gen AI for PM series- Part 1: From Gantt charts to Generative AI: How
+  Agentic AI is revolutionizing project management"
+date: 2025-08-27T15:04:35.590Z
+author: Dinesh R Singh, Nisha Rajput, Varsha Shekhawat
+authorimage: /img/blog-image.jpeg
+disable: false
+tags:
+  - Agentic AI
+  - Project Manager
+  - Scurm Master
+  - Gen AI for PM
+  - Gantt chart
+---
+<style>
+
+li {
+
+   font-size: 27px;
+
+   line-height: 33px;
+
+   max-width: none;
+
+}
+
+</style>
+
+## **Introduction: From static plans to intelligent partners**
+
+Picture this: You’re a project manager (PM) juggling deadlines, budget limits, and a flood of emails. You’ve got your trusted Gantt chart open, but within hours, it’s already outdated. A supplier delay changes Task A, which cascades into Tasks B and C, and suddenly your neat timeline looks like spaghetti. Sound familiar?
+
+For decades, project management has relied on **static tools** — from sticky notes on whiteboards to spreadsheets and Gantt charts. They’ve helped PMs visualize tasks and dependencies, but they’re fundamentally **snapshots in time**. The moment reality shifts, the tools fall behind, and project managers are left manually updating and chasing information.
+
+Today, a new paradigm is emerging. **Agentic AI** — intelligent agents that adapt, predict, and act — is turning project management from reactive firefighting into proactive leadership. Think of it not as another tool, but as your **digital co-pilot** who keeps the plan alive, responsive, and self-updating.
+
```

Comment:
I recommend adding in the following text prior to your example:

"In this blog post, we'll compare and contrast the differences, as shown in the example below, between traditional project management and project management that is optimized through the use of Agentic AI."